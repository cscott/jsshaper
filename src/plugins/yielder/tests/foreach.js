var log = (typeof console !== "undefined") && console.log || print;
(typeof require !== 'undefined') && require('setup.js');

// for each statement

for each (var i in [4,5,6,'foo']) {
    log(i);
}
