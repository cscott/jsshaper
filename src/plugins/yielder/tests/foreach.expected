4
5
6
foo
